mlos_bench/mlos_bench/environment/remote/os_env.py

```diff
@@ -19,7 +24,43 @@ class OSEnv(Environment):
     OS Level Environment for a host.
     """
 
-    def setup(self, tunables: TunableGroups, global_config: dict = None) -> bool:
+    def __init__(self,
+                 name: str,
+                 config: dict,
+                 global_config: Optional[dict] = None,
+                 tunables: Optional[TunableGroups] = None,
+                 service: Optional[Service] = None):
+        # pylint: disable=too-many-arguments
+        """
+        Create a new environment for remote execution.
+
+        Parameters
+        ----------
+        name: str
+            Human-readable name of the environment.
+        config : dict
+            Free-format dictionary that contains the benchmark environment
+            configuration. Each config must have at least the "tunable_params"
+            and the "const_args" sections.
+            `RemoteEnv` must also have at least some of the following parameters:
+            {setup, run, teardown, wait_boot}
+        global_config : dict
+            Free-format dictionary of global parameters (e.g., security credentials)
+            to be mixed in into the "const_args" section of the local config.
+        tunables : TunableGroups
+            A collection of tunable parameters for *all* environments.
+        service: Service
+            An optional service object (e.g., providing methods to
+            deploy or reboot a VM, etc.).
+        """
+        super().__init__(name, config, global_config, tunables, service)
```

Comment:
Once we add that `*` in the `Environment` `def __init__(self, *, ...):`, here we'll have to write:
```suggestion
        super().__init__(name=name, config=config, global_config=global_config, tunables=tunables, service=service)
```